src/connection.js

```diff
@@ -747,7 +779,11 @@ class Connection extends EventEmitter {
   saveTransaction(callback, name) {
     const transaction = new Transaction(name);
     if (this.config.options.tdsVersion < '7_2') {
-      return this.execSqlBatch(new Request('SAVE TRAN ' + transaction.name, callback));
+      const self = this;
+      return this.execSqlBatch(new Request('SAVE TRAN ' + transaction.name, function() {
```

Comment:
Done. Pushed an update.

I made the change and discovered arrow functions don't have implicit 'arguments'. Added the rest parameters to the arrow function, like '(...args) => {'.

One day I hope not to be surprised by JavaScript anymore :)